File: Java Word Examples/ParagraphsApiDemo.java
import spire.cloud.word.sdk.client.*;
import spire.cloud.word.sdk.client.api.ParagraphsApi;
import spire.cloud.word.sdk.client.model.*;
import java.util.List;

public class ParagraphsApiDemo {
    static String appId = "your id";
    static String appKey = "your key";
    static String baseUrl = "https://api.e-iceblue.cn";
    static Configuration wordConfiguration = new Configuration(appId, appKey, baseUrl);
    static ParagraphsApi paragraphsApi = new ParagraphsApi(wordConfiguration);

    public static void addParagraph () throws ApiException {
        String fileName = "addParagraph.docx";
        String name = fileName;
        String nodePath = "Section/0/Body/0";
        Integer indexOfParagraph = 1;
        String folder = "input";
        String password = null;
        String text = "E-iceblue";
        String destFilePath = "output/AddParagraph_output.docx";
        String storage = null;
        paragraphsApi.addParagraph(name, nodePath, destFilePath, folder, storage, indexOfParagraph, password, text);
    }
    public static void deleteParagraph() throws ApiException {
        String fileName = "deleteParagraph.docx";
        String name = fileName;
        String nodePath = "Section/0/Body/0";
        Integer index = 0;
        String folder = "input";
        String password = null;
        String storage = null;
        String destFilePath = "output/DeleteParagraph_output.docx";
        paragraphsApi.deleteParagraph(name, nodePath, index, destFilePath, folder, storage, password);
    }
    public static void getParagraphChildObjects() throws ApiException {
        String fileName = "getParaChildObjects.docx";
        String name = fileName;
        String nodePath = "Section/0/Body/0";
        Integer index = 0;
        String folder = "input";
        String storage = null;
        String password = null;
        List<ObjectInfo>  response = paragraphsApi.getParagraphChildObjects(name, nodePath, index, folder, storage,password);
    }
    public static void getParagraphformat() throws ApiException {
        String fileName = "getParagraphformat.docx";
        String name = fileName;
        String nodePath = "Section/0/Body/0";
        Integer index = 0;
        String folder = "input";
        String storage = null;
        String password = null;
        ParagraphFormat response = paragraphsApi.getParagraphFormat(name, nodePath, index, folder, storage,password);
    }
    public static void getParagraphs() throws ApiException {
        String fileName = "getParagraphs.docx";
        String name = fileName;
        String folder = "input";
        String storage = null;
        String password = null;
        String nodePath = null;
        List<ObjectInfo> response = paragraphsApi.getParagraphs(name, folder, storage, password, nodePath);
    }
    public static void updateParagraphFormat() throws ApiException {
        String fileName = "updateParagraphFormat.docx";
        String name = fileName;
        String nodePath = "Section/0/Body/0";
        Integer index = 0;
        String folder = "input";
        String storage = null;
        String password = null;
        ParagraphFormat format = new ParagraphFormat();
        format.setHorizontalAlignment(ParagraphFormat.HorizontalAlignmentEnum.DISTRIBUTE);
        format.setLeftOrInnerIndent(20f);
        format.setRightOrOutsideIndent(10f);
        format.setMirrorIndents(false);
        format.setFirstLineIndent(30f);
        format.setBackgroundColor( new Color(0, 200, 200));
        format.setBeforeSpacing(40f);
        format.setAfterSpacing(40f);
        format.setBeforeAutoSpacing(false);
        format.setBeforeAutoSpacing(false);
        format.setLineSpace(50f);
        format.setLineSpacingRule( ParagraphFormat.LineSpacingRuleEnum.EXACTLY);
        format.setWordWrap(true);
        Border topBorder = new Border();
        topBorder.setBorderType(Border.BorderTypeEnum.DOUBLE);
        Borders borders = new Borders();
        borders.setTopBorder(topBorder);
        format.setBorders(borders);
        String destFilePath = "output/UpdateParagraphFormat_output.docx";
        paragraphsApi.updateParagraphFormat(name, nodePath, index, format, destFilePath,folder, storage, password );
    }
    public static void updateParagraphText() throws ApiException {
        String fileName = "updateParagraphText.docx";
        String name = fileName;
        String nodePath = "Section/0/Body/0";
        Integer index = 0;
        String text = "E-iceblue";
        String folder = "input";
        String password = null;
        String storage = null;
        String destFilePath = "output/UpdateParagraphText_output.docx";
        paragraphsApi.updateParagraphText(name, nodePath, index, text, destFilePath,folder, storage, password);
    }
 }

File: Java Word Examples/ParagraphsApiDemo.cs
using Spire.Cloud.Word.Sdk.Api;
using Spire.Cloud.Word.Sdk.Client;
using Spire.Cloud.Word.Sdk.Model;
using System;

namespace CloudWord
{
    class ParagraphsApiDemo
    {
        static string appId = "your id";
        static string appKey = "your key";
        static string baseUrl = "https://api.e-iceblue.cn";
        static Configuration wordConfiguration = new Configuration(appId, appKey, baseUrl);
        static ParagraphsApi paragraphsApi = new ParagraphsApi(wordConfiguration);
        public static void AddParagraph()
        {         
            var fileName = "addParagraph.docx";
            string name = fileName;
            string nodePath = "Section/0/Body/0";
            int indexOfParagraph = 1;
            string folder = "input";
            string password = null;
            string text = "E-iceblue";
            string destFilePath = "output/AddParagraph_output.docx";
            string storage = null;
            paragraphsApi.AddParagraph(name, nodePath, destFilePath, folder, storage, indexOfParagraph, password, text);
        }
        public static void DeleteParagraph()
        {
            var fileName = "deleteParagraph.docx";
            string name = fileName;
            string nodePath = "Section/0/Body/0";
            int? index = 0;
            string folder = "input";
            string password = null;
            string storage = null;
            string destFilePath = "output/DeleteParagraph_output.docx";
            paragraphsApi.DeleteParagraph(name, nodePath, index, destFilePath, folder, storage, password);
        }
        public static void GetParagraphChildObjects()
        {
            var fileName = "getParaChildObjects.docx";
            string name = fileName;
            string nodePath = "Section/0/Body/0";
            int? index = 0;
            string folder = "input";
            string storage = null;
            string password = null;   
            var response = paragraphsApi.GetParagraphChildObjects(name, nodePath, index, folder, storage,password);
        }
        public static void GetParagraphformat()
        {
            var fileName = "getParagraphformat.docx";
            string name = fileName;
            string nodePath = "Section/0/Body/0";
            int? index = 0;
            string folder = "input";
            string storage = null;
            string password = null; 
            var response = paragraphsApi.GetParagraphFormat(name, nodePath, index, folder, storage,password);
        }
        public static void GetParagraphs()
        {
            string fileName = "getParagraphs.docx";
            string name = fileName;
            string folder = "input";
            string storage = null;
            string password = null;
            string nodePath = null;
            var response = paragraphsApi.GetParagraphs(name, folder, storage, password, nodePath);
        }
        public static void UpdateParagraphFormat()
        {
            var fileName = "updateParagraphFormat.docx";
            string name = fileName;
            string nodePath = "Section/0/Body/0";
            int? index = 0;
            string folder = "input";
            string storage = null;
            string password = null;
            ParagraphFormat format = new ParagraphFormat
            {
                HorizontalAlignment = ParagraphFormat.HorizontalAlignmentEnum.Distribute,
                LeftOrInnerIndent = 20,
                RightOrOutsideIndent = 10,
                MirrorIndents = false,
                FirstLineIndent = 30,
                BackgroundColor = new Color(0, 200, 200),
                BeforeSpacing = 40,
                AfterSpacing = 40,
                BeforeAutoSpacing = false,
                AfterAutoSpacing = false,
                LineSpace = 50,
                LineSpacingRule = ParagraphFormat.LineSpacingRuleEnum.Exactly,
                WordWrap = true,
                Borders = new Borders(
                    new Border(
                        Border.BorderTypeEnum.Double,
                        new Color(255, 10, 10),
                        true,
                        2))
            };
            string destFilePath = "output/UpdateParagraphFormat_output.docx";
            paragraphsApi.UpdateParagraphFormat(name, nodePath, index, format, destFilePath,folder, storage, password );
        }
        public static void UpdateParagraphText()
        {
            var fileName = "updateParagraphText.docx";
            string name = fileName;
            string nodePath = "Section/0/Body/0";
            int? index = 0;
            string text = "E-iceblue";
            string folder = "input";
            string password = null;
            string storage = null;
            string destFilePath = "output/UpdateParagraphText_output.docx";
            paragraphsApi.UpdateParagraphText(name, nodePath, index, text, destFilePath,folder, storage, password);
        }
    }
}
